internal/provider/resource_environment.go

```diff
@@ -223,6 +223,34 @@ func resourceEnvironment() *schema.Resource {
 					},
 				},
 			},
+			"repositories": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"name": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"database_type": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"allow_provisioning": {
+							Type:     schema.TypeBool,
+							Required: true,
+						},
+						"is_staging": {
```

Comment:
Is there a scenario where this isn't true?

Reply:
Same as above never seen this as true.

Reply:
Same answer as above. :)